src/EditorFeatures/Core/Shared/Extensions/SymbolDisplayPartExtensions.cs

```diff
@@ -27,14 +28,14 @@ public static Run ToRun(this SymbolDisplayPart part, IClassificationFormatMap fo
             return run;
         }
 
-        public static TextBlock ToTextBlock(this ImmutableArray<SymbolDisplayPart> parts, ClassificationTypeMap typeMap)
+        public static TextBlock ToTextBlock(this ImmutableArray<SymbolDisplayPart> parts, ClassificationTypeMap typeMap, TextWrapping textWrapping = TextWrapping.NoWrap)
```

Comment:
I'd rather not have optional arguments on internal APIs.  Instead I'd rather the caller have to think about it.  There aren't very many callers of these, can you just update them and make this required?